frontend/src/modules/APIVersion/utils.ts

```diff
@@ -5,9 +5,9 @@ export const isLowerOrEqualCurrentAPIVersion = (
   version: string,
   currentAPIVersion?: APIVersion,
 ) => {
-  if (currentAPIVersion === undefined) {
+  try {
+    return semver.lte(version, currentAPIVersion?.version as string)
```

Comment:
```suggestion
    return semver.lte(version, semver.clean(currentAPIVersion?.version) as string)
```
To handle version like `1.1.1+dev`